examples/playground/json-examples/3d-tiles-new-york.json

```diff
@@ -0,0 +1,30 @@
+{
+  "description": "The deck.gl website tile-3d-layer example",
+  "websiteUrl": "https://deck.gl/#/examples/geo-layers/tile-3d-layer",
+  "initialViewState": {
+    "latitude": 40.73380895124223,
+    "longitude": -73.99557610230511,
+    "pitch": 45,
+    "maxPitch": 60,
+    "bearing": 0,
+    "minZoom": 2,
+    "maxZoom": 30,
+    "zoom": 10.766728129551447
+  },
+  "views": [
+    {
+      "type": "MapView",
+      "mapStyle": "mapbox://styles/mapbox/dark-v9",
+      "controller": true
+    }
+  ],
+  "layers": [
+    {
+      "type": "Tile3DLayer",
+      "id": "tiles-new-york",
+      "DracoLoader": "DracoLoader",
```

Comment:
gltf parser doesn't work with `DracoWorkerLoader` @ibgreen

Reply:
Yes this needs to be fixed in loaders.gl. The new GLTFParser being developed for loaders.gl v2 should handle it but still has some bug.